mobile_verifier/migrations/15_speedtests_one_to_one.sql

```diff
@@ -0,0 +1,22 @@
+
+CREATE TABLE speedtests_migration (
+       pubkey text NOT NULL,
+       upload_speed bigint,
+       download_speed bigint,
+       latency integer,
+       serial text,
```

Comment:
Any issues with this column name being a keyword in Postgres? Should it be named `serial_num` or something?

Reply:
Never even noticed this, but all worked fine on local testing.   Still, prob best to rename and avoid

Reply:
addressed here: 67effc3558d2cd2c56ef07956dba0b19b02ffd95